example.rsf

```diff
@@ -0,0 +1,184 @@
+BasicInfo:
```

Comment:
See my review comment about this `.rsf` file.

Reply:
I'm unable to see any review comment on the PR thread. Try reposting it or commenting it here.

Reply:
Basically: we don’t have any standard way to build `.cia` files yet, nor are we planning on having one. Designing support for `cargo-3ds` would (most likely) be the way to do it in the future. Moreover, our repository is a library and not a released application, so we won’t need a `.rsf` file here (counting it also has some problems with how its setup of which I won’t go into detail).

Reply:
That makes perfect sense. I had it there for building and testing the examples differently than a .3dsx file. I will remove them and possible open an issue on cargo-3ds for possible different output formats.